requirements-devel.in

```diff
@@ -2,3 +2,4 @@
 -r tests/requirements.in
 pyflakes
 pep8
+certifi>=2024.07.04
```

Comment:
It's probably not needed here, as far as I can tell `requirements-devel.txt` reads `requirements.in` so it's enough to specify the version there

Reply:
Or, now that I think about it, do we need to specify the version in the `.in` files at all? If it's just a security fix, not an API incompatibility, then updating the pinned version in the `.txt` files is enough

Reply:
You are right. I was worried that when I would not specify it in the .in files then it would be overwritten with the original version on pip compile, but seems that the change is staying there. So I updated this to update the versions only in the required .txt files

Reply:
Yeah, `pip-compile` by default respects the versions present in requirements.txt to avoid bumping versions when you don't want to do that. To bump a specific dependency, you can run e.g. `pip-compile -P certifi ...`